internal/provider/resource_build_test.go

```diff
@@ -215,9 +215,9 @@ resource "apko_build" "foo" {
 `, repostr),
 			Check: resource.ComposeTestCheckFunc(
 				resource.TestCheckResourceAttr("apko_build.foo", "repo", repostr),
-				resource.TestCheckResourceAttr("apko_build.foo", "image_ref",
-					// With pinned packages we should always get this digest.
-					repo.Digest("sha256:c7ad87840041b6f27de750809ad1e211625fce06b9284f32be5e728fb25d8b50").String()),
+				//resource.TestCheckResourceAttr("apko_build.foo", "image_ref",
+				//	// With pinned packages we should always get this digest.
+				//	repo.Digest("sha256:c7ad87840041b6f27de750809ad1e211625fce06b9284f32be5e728fb25d8b50").String()),
```

Comment:
I wonder if this changed because of the inclusion of `t:0` in the `APKINDEX.tar.gz`.

Perhaps we should:
1. Publish the image somewhere public,
2. Update the digest with a comment to where we can find the old version to diff 😅

Reply:
That's a good theory.

Re-running a three-day-old test fails with the same digest: https://github.com/chainguard-dev/terraform-provider-apko/actions/runs/5049908003/jobs/9132237558#step:7:175

```
    resource_build_test.go:189: Step 1/1 error: Check failed: Check 2/3 error: apko_build.foo: Attribute 'image_ref' expected "127.0.0.1:37431/test@sha256:c7ad87840041b6f27de750809ad1e211625fce06b9284f32be5e728fb25d8b50", got "127.0.0.1:37431/test@sha256:2864ef148f9140f1aef312be95fd3cb222df900ed5d99c7447414eafaae31233"
```